src/main/java/frc/robot/Robot.java

```diff
@@ -93,17 +93,20 @@ public void robotPeriodic() {
 
     // Checks every 100 milliseconds (roughly 10 robot cycles) to see if any Spark Maxes have rebooted
     // if one has it will then rerun CAN ID configurations on it to stop CAN bus from overflowing
-
     if (m_timer.get() >= 5.0) {
       SparkMaxFactory.Companion.updateCanFramePeriods();
       m_timer.reset();
     }
+
+    // For testing purposes
+    m_robotContainer.getArmSupersystem().calculateArmAngleLimit(20);
+    m_robotContainer.getArmSupersystem().getDriveSpeed();
```

Comment:
These shouldn't be run when enabled

Reply:
Yes, I believe you fixed both of these, correct?